inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -408,6 +410,33 @@ void LayerTestsCommon::Validate() {
     Compare(expectedOutputs, actualOutputs);
 }
 
+std::string LayerTestsCommon::getKernelPrecision(const std::string& layerName) {
```

Comment:
As soon as the function returns runtime precision shouldn't we rename it accordingly `getRuntimePrecision`?

Reply:
Done, thanks